pkg/consts/consts.go

```diff
@@ -213,6 +213,9 @@ var (
 
 // load balancer
 const (
+	// After dual-stack is supported, all references should be updated.
+	DualstackSupported = false
```

Comment:
Could you elabrate why a const flag is required here? Dualstack is supported since k8s v1.20, and hence it should be supported already.

Reply:
Explained here: https://github.com/kubernetes-sigs/cloud-provider-azure/pull/3266#discussion_r1101117742